src/common/form/index.js

```diff
@@ -165,12 +165,12 @@ export class SizeInput extends BaseComponent {
   constructor (props) {
     super(props)
 
-    this.state = this._createStateFromBytes(firstDefined(props.value, props.defaultValue, 0))
+    this.state = this._createStateFromBytes(firstDefined(props.value, props.defaultValue))
```

Comment:
If `props.value` and `props.defaultValue` are undefined, I want the input field to be empty, which will not be the case if I call `this._createStateFromBytes` with `0`.